core/src/stores/store.rs

```diff
@@ -311,9 +380,41 @@ pub const POSTGRES_TABLES: [&'static str; 11] = [
        status                   TEXT NOT NULL,
        FOREIGN KEY(data_source) REFERENCES data_sources(id)
     );",
+    "-- databases
+    CREATE TABLE IF NOT EXISTS databases (
+       id                   BIGSERIAL PRIMARY KEY,
+       created              BIGINT NOT NULL,
+       database_id          TEXT NOT NULL, -- unique within data source
+       internal_id          TEXT NOT NULL, -- unique globally
+       data_source          BIGINT NOT NULL,
+       name                 TEXT NOT NULL,
```

Comment:
Agreed with your comments on slack that this can probably be removed from `core` I think we want to have the strict minimum viable set in `core` (at least to begin with) :+1:

Reply:
Sounds good. I can then remove the `upsert_database` store method and replace it with `register_database`.
Will remove "name" from db table (simpler to just have a "table_id" that serves as the name of the column in the DB). I am wondering about the decription though. Do we want it inside `core` or is it fine to have it in `front` ? 
The description will likely be used as part of the prompt to generate the SQL.

Reply:
Actually...

Description is definitely a `core` concern since we will output it as part of the DatabaseSchema call. But as a consequence name is as well. So let's keep it here :+1: